### PR TITLE
Add support for unencrypted load files

### PR DESCRIPTION
**Pull request by petter-fidesmo**
- Extend CcmInstall with encryptLoad field
- Make a generic installToLocalCard task config/action
- Add new task installToLocalCardUnencrypted

### CODE REVIEW

**Comment by petter-fidesmo** [comment]
@slomo What do you think of this?

File: src/main/groovy/com/fidesmo/gradle/plugin/FidesmoPlugin.groovy [hunk 1]
```diff
@@ -82,29 +82,35 @@ class FidesmoPlugin implements Plugin<Project> {
...
-            dependsOn(project.deleteFromLocalCard)
-
-            doLast {
-                def applicationAid
-                if(fidesmoExtension?.instanceAid) {
-                    applicationAid = new Aid(fidesmoExtension.instanceAid).hexString
-                } else {
-                    applicationAid = jcExtension.cap.applets.first().aid.hexString
+        def installToLocalCard = { encryptLoad ->
+            { it ->
```
Comment (slomo) [inline src/main/groovy/com/fidesmo/gradle/plugin/FidesmoPlugin.groovy]: The `it` parameter is always implicit unless you write `{ -> ... }`. It exists for iterator cases. Maybe we should name this parameter more descriptive (like `task`).

**Comment by slomo** [comment]
Other than a stylistic comment :+1:

File: src/main/groovy/com/fidesmo/gradle/plugin/FidesmoPlugin.groovy (hunk 1 above)
Comment (petter-fidesmo) [inline src/main/groovy/com/fidesmo/gradle/plugin/FidesmoPlugin.groovy]: Thanks, fixed!

**Pull request merged**